e2e/tests/nightly/gke-apps/rstudio.spec.ts

```diff
@@ -18,36 +19,21 @@ describe('RStudio GKE App', () => {
   test('Create and delete a RStudio GKE app', async () => {
     await findOrCreateWorkspace(page, { workspaceName });
 
-    const appsPanel = new AppsPanel(page);
-    await appsPanel.open();
+    const configPanel = new RStudioConfigurationPanel(page);
+    await configPanel.startRStudioGkeApp();
 
-    // RStudio is not running, so it appears in unexpanded mode
+    // 1. closes the config panel
+    // 2. waits a few seconds
+    // 3. opens the apps panel
 
-    const unexpandedRStudioXPath = `${appsPanel.getXpath()}//*[@data-test-id="RStudio-unexpanded"]`;
-    const unexpandedRStudio = new Button(page, unexpandedRStudioXPath);
+    await page.waitForXPath(configPanel.getXpath(), { visible: false });
 
-    expect(await unexpandedRStudio.exists()).toBeTruthy();
-    await unexpandedRStudio.click();
-
-    // clicking RStudio expands it, exposing its buttons
+    const appsPanel = new AppsPanel(page);
+    await appsPanel.isVisible();
 
     const expandedRStudioXpath = `${appsPanel.getXpath()}//*[@data-test-id="RStudio-expanded"]`;
     await page.waitForXPath(expandedRStudioXpath);
 
-    const createXPath = `${expandedRStudioXpath}//*[@data-test-id="apps-panel-button-Create"]`;
-    const createButton = new Button(page, createXPath);
-    expect(await createButton.exists()).toBeTruthy();
-
-    const pauseXPath = `${expandedRStudioXpath}//*[@data-test-id="apps-panel-button-Pause"]`;
-    const pauseButton = new Button(page, pauseXPath);
-    expect(await pauseButton.exists()).toBeTruthy();
-
-    const launchXPath = `${expandedRStudioXpath}//*[@data-test-id="apps-panel-button-Launch"]`;
-    const launchButton = new Button(page, launchXPath);
-    expect(await launchButton.exists()).toBeTruthy();
```

Comment:
This E2E coverage was removed. I don't think it's worth verifying that these buttons exist in an E2E test unless we are also verifying their functionality.